src/libcore/lib.rs

```diff
@@ -82,6 +82,7 @@
 #![feature(const_fn)]
 #![feature(const_int_ops)]
 #![feature(const_fn_union)]
+#![feature(const_manually_drop_new)]
```

Comment:
This is not used?

Reply:
It is?

Reply:
Oh I see; maybe you should remove `#[rustc_const_unstable(feature = "const_manually_drop_new")]` and stabilize the `const fn`ness instead?

Reply:
Maybe it’s time to do that, but probably in another PR. This PR is intended to only affect unstable APIs so that it doesn’t need to go through FCP.